x-pack/plugins/ml/public/application/timeseriesexplorer/components/forecasting_modal/forecasting_modal.js

```diff
@@ -546,4 +557,4 @@ export class ForecastingModalUI extends Component {
   }
 }
 
-export const ForecastingModal = withKibana(ForecastingModalUI);
+export const ForecastingModal = ForecastingModalUI;
```

Comment:
`ForecastingModalUI` isn't needed anymore now. The class can just called `ForecastingModal`

Reply:
Updated in 70a4278a495651f8acb2d17f57cf79570ac71c26